server/migrations/20190318001100-addUuidExt.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  up: async function (queryInterface) {
+    return queryInterface.sequelize.query(`CREATE EXTENSION "uuid-ossp"`);
```

Comment:
Why this instead of the previous way? If we are using ``uuid_generate_v4``, should we have another story later to convert previous tables to use this as well?